.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           curl --proto '=https' --tlsv1.2 --retry 10 --retry-connrefused -fsSL "https://sh.rustup.rs" | sh -s -- --profile minimal --default-toolchain none -y
           echo "${CARGO_HOME:-$HOME/.cargo}/bin" >> $GITHUB_PATH
       - name: Install SQLFluff
-        run: pip install sqlfluff==2.1.4
+        run: pip install sqlfluff==2.3.0
```

Comment:
I can't comment at the correct line, but: do we still need --all-features in the CI build strategy? all-features is the same as pro now?